UIRes/pluginmetadata.json

```diff
@@ -0,0 +1,77 @@
+[
+  {
+	"Name": "AccurateCountDown",
+    "DevSupportState": "Obsolete",
+    "DevSupportStateReason": "Replaced by EngageTimer"
+  },
+  {
+    "Name": "ACTEndcounter",
+    "DevSupportState": "Adoptable"
+  },
+  {
+    "Name": "BrowserHost.Plugin",
+    "DevSupportState": "Obsolete",
+    "DevSupportStateReason": "Replaced by Browsingway"
+  },
+  {
+    "Name": "CharSyncPlugin",
+    "DevSupportState": "Obsolete",
+    "DevSupportStateReason": "Replaced by Character Data Sync"
+  },
+  {
+    "Name": "ChatExtender",
```

Comment:
We should probably note that ChatExtender was actually broken up into separate plugins.
- Chat bubbles are handled by Chat Bubbles (Haplo)
- Chat translation is handled by Chat Translator (Haplo)
- Chat logging was replaced by XIVLogger (Cadaeix)